cpp/core/operators/c2r/ColumnarToRow.h

```diff
@@ -17,119 +17,36 @@
 
 #pragma once
 
-#include <arrow/array/array_base.h>
-#include <arrow/array/array_binary.h>
-#include <arrow/array/array_primitive.h>
-#include <arrow/buffer.h>
-#include <arrow/record_batch.h>
-#include <arrow/type.h>
-#include <arrow/util/bit_util.h>
 #include "memory/ColumnarBatch.h"
 
-#include <boost/align.hpp>
-
 namespace gluten {
 
 class ColumnarToRowConverter {
  public:
-  explicit ColumnarToRowConverter() {}
```

Comment:
Why remove `explicit` keyword?

Reply:
It should be used for 1 argument constructor.
https://www.scaler.com/topics/cpp-explicit/
If we have a class that has a constructor with a single argument